4-fleetscope/modules/env_baseline/variables.tf

```diff
@@ -14,3 +14,29 @@
  * limitations under the License.
  */
 
+variable "env" {
+  description = "The environment to prepare (ex. development)"
+  type        = string
+}
+
+variable "project_id" {
+  description = "The fleet project ID"
+  type        = string
+}
+
+variable "scope_id" {
+  description = "The fleet scope ID"
+  type        = string
+  default     = "REGULAR"
+}
+
+variable "membership_id" {
+  description = "The membership ID in the scope"
+  type        = string
+}
+
+variable "namespace_id" {
+  description = "The fleet namespace ID"
+  type        = string
+  default     = "REGULAR"
+}
```

Comment:
Running `make docker_generate_docs` will update the `modules/env_baseline` README.md with these new input variables.

Reply:
added README.md